_posts/2019-05-28-diamond.md

```diff
@@ -0,0 +1,11 @@
+---
+site: freiburg
+title: "Visit from Macromolecular Crystallography Group"
+tags: ???
```

Comment:
no idea what should go here

Reply:
You can grep for `tags:` in the post directory to find out a list of options. Probably:

```
tags: [tools]
```

Reply:
The problem was that none of the tags really seemed to fit. But I guess tools works as well as anything else, thanks.

Reply:
Is the tool in galaxy? Can you link to it? Tags are whatever you want them to be. a couple have special icons, but if you think there's a more appropriate tag, just apply it. E.g. `visit` could be fine, or `collaboration`

Reply:
Ah okay. Then `collaboration` is probably indeed better.